reference/upgrades.md

```diff
@@ -83,7 +84,7 @@ To view the logs of upgrade job, execute:
 
 {% tab title="Command" %}
 ```text
-kubectl -n <namespace> job/job-name
+kubectl get logs job/job-name -n <namespace>
```

Comment:
Does this command work?
I thought it has to be,
kubectl logs -f <upgrade-job-pod-name> -n <mayastor-namespace>

@sinhaashish @niladrih ?

Reply:
Yes! `-f` is optional, but yeah.

Reply:
kubectl logs upgrade-job-pod-name -n mayastor-namespace